airflow/www/views.py

```diff
@@ -1024,12 +1024,12 @@ def blocked(self, session=None):
             )
             if 'all_dags' not in filter_dag_ids:
                 dags = dags.filter(DR.dag_id.in_(filter_dag_ids))
-            dags = dags.all()
```

Comment:
Why did you change it? This should work because all is a method of the query class that retrieves all items that are filtered in the current query. This work differently from "one", which only retrieves first. https://docs.sqlalchemy.org/en/13/orm/query.html#sqlalchemy.orm.query.Query.all